common/survey/_internal/surveyInfo.js

```diff
@@ -84,6 +84,14 @@ const hasCollectReportIssues = R.pipe(
 
 const isFromCollect = R.pipe(getCollectUri, R.isNil, R.not)
 
+const getLanguage = preferred => surveyInfo => {
```

Comment:
can ```preferred``` be called something else? It's not clear what it refers to. maybe ```preferredLang``` ?

Reply:
Renamed